app/controllers/claims/support/schools/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Claims::Support::Schools::UsersController < Claims::Support::ApplicationController
+  before_action :set_school
+
+  def index
+    @users = @school.users
```

Comment:
I noticed this when thinking about replicating this for placements.
This should be:
```suggestion
    @users = @school.users.claims
```